python/settings.py

```diff
@@ -1,8 +1,10 @@
+from typing import List
+
 class Settings(object):
 
     # HALT! do NOT change this without changing corresponding type in the frontend! <----
     # Also note that this uses camelCase because that is standard in JS frontend
-    def __init__(self, showGlobalVars=True, default_filter_vars=[], default_filter_types=[], *args, **kwargs):
+    def __init__(self, showGlobalVars=True, default_filter_vars: List[str] = [], default_filter_types: List[str] = [], *args, **kwargs):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/manual/almenon214/AREPL-backend/pullRequest?prid=5181101)